### PR TITLE
Add fetch mock

### LINKED ISSUE #77: Add fetch mock to service-worker-mock
Thanks for a great mocking tool!

Is there any reason why you don't provide a simple mock/swallow function for `fetch`? I was on my way giving up this package (thanks God I didn't) cause I couldn't find how I was supposed to handle the lack of `fetch` function. Maybe it's just that I'm rarely stupid. Either way I think it would be great if everything worked out of the box.

### PR DESCRIPTION
**Pull request by kontrollanten**
Closes #77

### CODE REVIEW

File: packages/service-worker-mock/models/Request.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ class Request extends Body {
   }
 
   clone() {
-    if (this.bodyUsed) throwBodyUsed('json');
```
Comment (zackargyle) [inline packages/service-worker-mock/models/Request.js]: Is this change related?

**Comment by zackargyle** [comment]
Thanks for this! Just the once question.

File: packages/service-worker-mock/models/Request.js (hunk 1 above)
Comment (kontrollanten) [reply packages/service-worker-mock/models/Request.js]: No, it was something I saw while going through the code. So I made an additional commit to clean it up, cause I guess it has no effect?

Comment (zackargyle) [reply packages/service-worker-mock/models/Request.js]: Looks like you're right! Good catch :)

**Comment by zackargyle** [comment]
Thanks again. Appreciate the contribution!
